amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -3760,3 +3911,102 @@ definitions:
           - SiteCollaborator
           - SiteContributor
           - SiteManager
+  EventsInfo:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventInfo'
+  EventInfo:
+    type: object
+    required:
+      - id
+      - name
+      - type
+    properties:
+      id:
+        type: string
+        description: this is the id of the event
+      name:
+        type: string
+        description: This is the display label of the event
+      type:
+        type: string
+        description: this is event type
+  EventBodyCreate:
+    type: object
+    required:
+      - title
+    properties:
+      name:
+        type: string
+        description: This is the display label of the event
+      type:
+        type: string
+        description: this is event type
+        default: Simple
+  EventEntry:
+    type: object
+    required:
+      - name
+      - type
+    properties:
+      name:
+        type: string
+        description: This is the display label of the event
+      type:
+        type: string
+        description: this is event type
+  EventBodyUpdate:
+    type: object
+    required:
+      - id
+    properties:
+      name:
```

Comment:
Is this missing the id?

Reply:
Done

Reply:
Not done :(

Reply:
id is not required here so we already removed it from required indentation

Reply:
Please can you check some other PUT endpoints? From memory then I think the framework always requires the id in the body, even though it's supplied in the URI.

Reply:
We discussed this one in some detail and I was mistaken.  The framework requires the id in the path.